common/archiver/filestore/visibilityArchiver.go

```diff
@@ -336,11 +336,9 @@ func convertToExecutionInfo(record *archiverproto.ArchiveVisibilityRequest) *wor
 		Type: &commonpb.WorkflowType{
 			Name: record.WorkflowTypeName,
 		},
-		StartTime: &types.Int64Value{
-			Value: record.StartTimestamp},
-		ExecutionTime: record.ExecutionTimestamp,
-		CloseTime: &types.Int64Value{
-			Value: record.CloseTimestamp},
+		StartTime:     timestamp.TimePtr(timestamp.UnixOrZeroTime(record.StartTimestamp)),
```

Comment:
Do we need to need to convert timestamp fields on archiverproto.ArchiveVisibilityRequest to protoTime?  I guess then this conversion will go away.

Reply:
My guess is @shawnhathaway will take care of this with his change to convert all internal API to protoTime instead of timestamps.